app/views/admin/invoices/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="invoices-header">
+  <h2> Invoices </h2>
+</div>
+<div class="invoices">
+  <% @invoices.each do |invoice| %>
+    <div class="invoice">
+      <%= link_to invoice.id, admin_invoice_path(invoice) %>
```

Comment:
In the wireframe, it shows each invoice number has `Invoice #` before each invoice id/link. I think you should add that.

Here's how I wrote the HTML for the Merchant Invoices Index page, which is very similar visually.

```
<div id="merchant_invoices">
  <h1>My Invoices</h1>
  <% @merchant.distinct_invoices.each do |invoice| %>
    <p>Invoice #<%= link_to "#{invoice.id}", merchant_invoice_path(@merchant, invoice), id: "invoice#{invoice.id}_link" %></p>
  <% end %>
</div>
```

![image](https://github.com/z-fitch/little-shop-7/assets/8324716/9d131cf1-45ec-44e0-9363-2a4d063cac2d)

Reply:
fixed